### DISCUSSION TITLE
advice for capstone younglings

### DISCUSSION THREAD

**Discussion by d-opri** [neuefische/questions#458] [Web]
moin moin everyone, 

we just started our capstone phase yesterday, and i thought it would be great to get some input from the ones who have gone through this process already, or are finishing up their projects. any advice you would have loved to know when you started? things you would have done differently now that you know more? any precautions to take? or just what you learned from this experience

i would appreciate any input and counsel! 
take care

**Comment by ufelgen** [comment]
Hi Dani, I would say first of all, try not to get too overwhelmed when things seem difficult. 
Split the project in small and appropriate steps (= don't write giant user stories that take several days to conquer) and stick to those. Try to focus on one thing at a time (one user story) and don't get sidetracked. Keep the big picture in mind (I'm sure you have a concept in your head of what your app should look like in the end) but don't think you have to solve it all at once - small steps that are executed well will in the end result in a great app, I'm sure! 
Good luck, Ulli

**Reply by d-opri** [reply]
thank you for your reply! i will definetly make my next userstory more digestible... 

thankd again and have a great day
